### PR TITLE
Enhance Card Image Responsiveness on Small Screens (#327)

### LINKED ISSUE #327: ENHANCEMENT: Card Image Covers Large Screen Area on Small Devices
### Is there an existing issue raised for this?

- [X] I have searched the existing issues

### Describe the improvement

The recipe card image currently takes up too much space on small screen devices, making the layout appear unbalanced and cluttered.

Uploading 23.10.2024_20.59.39_REC.mp4…

### Component

On smaller screen sizes (e.g., mobile devices), the image should resize appropriately to fit the screen, maintaining a balanced layout with the profile details.

### It should be implemented because

image should resize appropriately to fit the screen, maintaining a balanced layout with the profile details.

### Would you like to work on this issue?

None

### Terms & Conditions

- [X] I agree to follow the Code of Conduct
- [X] I'm a GSSOC'24 Ext contributor

### PR DESCRIPTION
**Pull request by dipexplorer**
What does this PR do?
This PR improves the responsiveness of the card image on small screens to maintain a balanced layout, fixing layout issues on mobile devices where the image covers excessive space.
Fixes #327.

How should this be tested?
Test 1: Resize browser window below 750px width and verify the card image resizes proportionately to maintain a balanced layout.
Test 2: Check that the image and other card elements adjust without causing overflow or unbalanced space on smaller screens.

Checklist
- [x]  Filled out the "How to test" section in this PR
- [x]  Read Contributing guide
- [x]  Self-reviewed my own code
- [x]  Checked for warnings, there are none
- [x]  Removed all console.logs
- [x]  The changes don’t cause any responsiveness issues

### CODE REVIEW

**Comment by dipexplorer** [comment]
Hi @AlfiyaSiddique ,

I have completed the necessary changes and requested reviews, but it appears that the pull request is blocked automaticaly. I want to ensure I follow the proper procedures and comply with the repository's rules.

Could you please provide guidance on what steps I should take to move forward? or is there anything else I might need to address to get this PR approved?

File: frontend/index.html [hunk 1]
```diff
@@ -16,6 +16,15 @@
...
+<style>
+  /* responsive style for recipe images */
+  @media (max-width: 768px) {
+      .recipe-card-img {
+          height: 9rem;
+      }
+  }
+</style>
+
 <body>
```
Comment (AlfiyaSiddique) [inline frontend/index.html:28]: @dipexplorer put this in some CSS file

**Comment by AlfiyaSiddique** [comment]
@dipexplorer one comment

**Comment by AlfiyaSiddique** [comment]
And @dipexplorer how did you approve changes in issue #310 ??

**Comment by dipexplorer** [comment]
> And @dipexplorer how did you approve changes in issue #310 ??

 I approved the PR unconditionally by mistake. Sorry for the oversight—I’ll be more careful moving forward.

**Comment by AlfiyaSiddique** [comment]
Also please update the PR

File: frontend/index.html (hunk 1 above)
Comment (AlfiyaSiddique) [reply frontend/index.html:28]: @dipexplorer please fix this

**Comment by dipexplorer** [comment]
> Also please update the PR

@AlfiyaSiddique , 
Could you explain to me why my PR hasn't been merged yet?

File: frontend/index.html (hunk 1 above)
Comment (AlfiyaSiddique) [reply frontend/index.html:28]: @dipexplorer  ???

**Comment by AlfiyaSiddique** [comment]
@dipexplorer above I have asked you to move the styles to CSS file that you have not done yet that's why!
image

**Review by dipexplorer — COMMENTED** [review]
fixing css conflict

**Comment by AlfiyaSiddique** [comment]
@dipexplorer please update your pr there are new changes which is not reflecting on this PR

**Comment by AlfiyaSiddique** [comment]
Like the footer

**Pull request closed without merging**